core/src/main/java/org/apache/iceberg/MetadataUpdate.java

```diff
@@ -250,6 +251,8 @@ public SetSnapshotRef(String name, Long snapshotId, SnapshotRefType type, Intege
       this.maxRefAgeMs = maxRefAgeMs;
     }
 
+    // TODO - Consider renaming to `refName`, as the JSON is called `ref`. This also matches the equivalent
+    //  UpdateRequirement.
     public String name() {
```

Comment:
TODO / Possible Follow Up - Consider renaming to `refName`, as the JSON is called `ref`. This also matches the equivalent UpdateRequirement.

Reply:
I did update this to `refName`, as we updated the spec to refer to this field as `ref-name` instead of simply `name`.

I can revert the changes to the code though.

Reply:
Change tot he code has been reverted.